core-services/image-mirroring/_config.yaml

```
@@ -477,6 +477,8 @@ supplementalCIImages:
     image: quay.io/devtools_gitops/go-toolset:1.25.5
   ci/openshift-gitops-go-toolset:1.25.9:
     image: quay.io/devtools_gitops/go-toolset:1.25.9
+  ci/openshift-gitops-go-toolset:1.25.9:
```

YAMLlint (1.38.0)

[error] duplication of key "ci/openshift-gitops-go-toolset:1.25.9" in mapping

(key-duplicates)